contrib/docker-entrypoint.sh

```diff
@@ -12,15 +12,14 @@ if [ $(id -u) = 0 ]; then
 fi
 
 if [ "$1" = 'test' ]; then
-  eval $CMD_PREFIX /home/pganalyze/collector --test --no-log-timestamps --no-reload
+  shift
+  eval $CMD_PREFIX /home/pganalyze/collector --test --no-log-timestamps --no-reload "$@"
+elif [ "$1" = 'test-explain' ]; then
+  shift
+  eval $CMD_PREFIX /home/pganalyze/collector --test-explain --no-log-timestamps "$@"
+elif  [ "$1" = 'collector' ]; then
+  shift
+  eval $CMD_PREFIX /home/pganalyze/collector --statefile=/state/pganalyze-collector.state --no-log-timestamps "$@"
+else
```

Comment:
Technically I don't think the `else` is necessary, since we are utilizing `exec` in the `CMD_PREFIX` always, but I guess this doesn't hurt.

Reply:
Yeah, I always felt "wait doesn't that last `eval $CMD_PREFIX "$@"` always called?" (which is not true with `exec` indeed), so I figured that this will help a bit readability.